src/lib/middleware.mjs

```diff
@@ -15,6 +15,16 @@ const DEFAULT_URL = "https://db.fauna.com";
 export function logArgv(argv) {
   const logger = container.resolve("logger");
   logger.debug(JSON.stringify(argv, null, 4), "argv", argv);
+  const faunaEnvVars = Object.entries(process.env)
+    .filter(([key]) => key.startsWith("FAUNA"))
+    .reduce((acc, [key, value]) => {
+      acc[key] = value;
+      return acc;
+    }, {});
+  logger.debug(
+    `Existing Fauna environment variables: ${JSON.stringify(faunaEnvVars)}`,
+    "argv",
+  );
```

Comment:
example output

```
...
[argv]: Existing Fauna environment variables: {"FAUNA_ACCOUNT_URL":"https://account.fauna.com"}
[argv]: Defaulted url to 'https://db.fauna.com' no --url was provided
...
```

Reply:
I moved this to only caputre the env vars if user is in debug. otherwise we're revving through env vars on every command.